exercises/K-functions-parameters/exercise4.js

```diff
@@ -1,5 +1,8 @@
-// Declare your function first
+function sum (a,b) {
+    return a + b;
+   
+}
 
-// Call the function and assign to a variable `sum`
+var result = sum(13, 124);
```

Comment:
var is acceptable, but I remember in the class, the teacher said is better to use let